internal/riverinternaltest/testfactory/test_factory.go

```diff
@@ -36,6 +36,14 @@ type JobOpts struct {
 func Job(ctx context.Context, tb testing.TB, exec riverdriver.Executor, opts *JobOpts) *rivertype.JobRow {
 	tb.Helper()
 
+	job, err := exec.JobInsertFull(ctx, JobBuild(tb, opts))
+	require.NoError(tb, err)
+	return job
+}
+
+func JobBuild(tb testing.TB, opts *JobOpts) *riverdriver.JobInsertFullParams {
```

Comment:
I was thinking about doing something like this before, but I think we need to establish a different convention, because as is, the naming suggests that this is a factory function for a "job build" entity.

Maybe something like `Job_Build` or `Job_InsertParams`?

Reply:
renamed to `Job_Build`